packages/framework/src/handler.ts

```diff
@@ -57,21 +57,29 @@ interface IActionResponse<TBody extends string = string> {
 
 export class NovuRequestHandler<Input extends any[] = any[], Output = any> {
   public readonly frameworkName: string;
-
   public readonly handler: Handler;
-
   public readonly client: Client;
-
   private readonly hmacEnabled: boolean;
+  private readonly http: AxiosInstance;
 
   constructor(options: INovuRequestHandlerOptions<Input, Output>) {
     this.handler = options.handler;
     this.client = options.client ? options.client : new Client();
     this.client.addWorkflows(options.workflows);
+    this.http = this.initHttpClient();
     this.frameworkName = options.frameworkName;
     this.hmacEnabled = this.client.strictAuthentication;
   }
 
+  private initHttpClient() {
+    return axios.create({
```

Comment:
There is no need for an extra dependency on the framework package that increases it's bundle and runtime size. Let's use fetch.